pkg/operation/botanist/component/apiserverproxy/apiserver_proxy.go

```diff
@@ -0,0 +1,588 @@
+								v1beta1constants.LabelNetworkPolicyShootFromSeed:    v1beta1constants.LabelNetworkPolicyAllowed,
+							},
+						),
+					},
+					Spec: corev1.PodSpec{
+						ServiceAccountName: serviceAcountName,
+						PriorityClassName:  "system-node-critical",
+						Tolerations: []corev1.Toleration{
+							{Effect: corev1.TaintEffectNoSchedule, Operator: corev1.TolerationOpExists},
+							{Effect: corev1.TaintEffectNoExecute, Operator: corev1.TolerationOpExists},
+						},
+						HostNetwork:                  true,
+						AutomountServiceAccountToken: pointer.Bool(false),
+						SecurityContext: &corev1.PodSecurityContext{
+							SeccompProfile: &corev1.SeccompProfile{
+								Type: corev1.SeccompProfileTypeRuntimeDefault,
+							},
+						},
+						InitContainers: []corev1.Container{
+							{
+								Name:            "setup",
+								Image:           a.values.APIServerProxySidecarImage,
+								ImagePullPolicy: corev1.PullIfNotPresent,
+								Args: []string{
+									fmt.Sprintf("--ip-address=%s", a.advertiseIPAddress),
+									"--setup-iptables=false",
+									"--daemon=false",
+									"--interface=lo",
+								},
+								SecurityContext: &corev1.SecurityContext{
+									Capabilities: &corev1.Capabilities{
+										Add: []corev1.Capability{
+											"NET_ADMIN",
+										},
+									},
+								},
+								Resources: corev1.ResourceRequirements{
+									Requests: corev1.ResourceList{
+										corev1.ResourceCPU:    resource.MustParse("20m"),
+										corev1.ResourceMemory: resource.MustParse("20Mi"),
+									},
+									Limits: corev1.ResourceList{
+										corev1.ResourceMemory: resource.MustParse("200Mi"),
+									},
+								},
+							},
+						},
+						Containers: []corev1.Container{
+							{
+								Name:            "sidecar",
+								Image:           a.values.APIServerProxySidecarImage,
+								ImagePullPolicy: corev1.PullIfNotPresent,
+								Args: []string{
+									fmt.Sprintf("--ip-address=%s", a.advertiseIPAddress),
+									"--setup-iptables=false",
+									"--interface=lo",
+								},
+								SecurityContext: &corev1.SecurityContext{
+									Capabilities: &corev1.Capabilities{
+										Add: []corev1.Capability{
+											"NET_ADMIN",
+										},
+									},
+								},
+								Resources: corev1.ResourceRequirements{
+									Requests: corev1.ResourceList{
+										corev1.ResourceCPU:    resource.MustParse("20m"),
+										corev1.ResourceMemory: resource.MustParse("20Mi"),
+									},
+									Limits: corev1.ResourceList{
+										corev1.ResourceMemory: resource.MustParse("90Mi"),
+									},
+								},
+							},
+							{
+								Name:            "proxy",
+								Image:           a.values.APIServerProxyImage,
+								ImagePullPolicy: corev1.PullIfNotPresent,
+								Command: []string{
+									"envoy",
+									"--concurrency",
+									"2",
+									"--use-dynamic-base-id",
+									"-c",
+									"/etc/apiserver-proxy/envoy.yaml",
```

Comment:
- define a constant for the volume mount path and use it here and in the volume mount
- define a constant for the data key in the config map and use it here and in the configmap